utils.py

```diff
@@ -304,7 +320,9 @@ def make_clean_vcf(vcf, path=None):
                 continue
             else:
                 GT = "0/1" if line_split[-1] == "./." else line_split[-1]
-                clean_vcf_handle.write('\t'.join([line_split[0], line_split[1], ".", line_split[3], line_split[4], ".", ".", ".", line_split[8], GT]) + '\n')
+                info_entries = [line_split[7].split('=')[0] for x in line_split[7].split(';')]
```

Comment:
What is this capturing? The iterated variable `x` does not seem to be used here.

Reply:
Good spot, that was a bug. In very few cases would it have any negative effect which is why it wasn't caught in tests